lib/capsize/application.rb

```diff
@@ -0,0 +1,31 @@
+module Capsize
+  module Application
+    def self.load_tasks
+      Application.load_tasks_in_isolation do
+        require 'capistrano/all'
+        cap = Capistrano::Application.new
+        Rake.application = cap
+        cap.init
+        cap.load_rakefile
+        cap.tasks
+      end
+    end
+
+    def self.load_tasks_in_isolation
+      read, write = IO.pipe
```

Comment:
This I left because I needed `read` and `pid` to exist outside of the `IO.pipe` block, though I can work around that if the block is better.